release-notes.md

```diff
@@ -10,6 +10,23 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
 
 [Sign up](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=Wht7-jR7h0OUrtLBeN7O4UJN9zAhIEhJr3PBfyMf9wdUQTI2S0pMVEVYS1k3UUNJVDNDWlRUTFk4Qi4u) for a free developer preview.
 
+## Aug 10, 2020
+
+### iOS Core 2.7.2 \(Released with ACPCore version 2.7.3 on Cocoapods\)
+
+* Fixed a crash happening on `AdobeMarketingMobile::RulesEngine::ProcessEventForRules()`.
+* Fixed an issue where null values in rules consequences were not respected.
+
+### iOS Identity 2.3.2
+
+* Identity shared state now gets updated in current session on server response changes for blob or locationHint.
+* In order to improve the Analytics push tracking reports, the push notification preferences (a.push.optin) are now forwarded to Analytics when `setPushIdentifier` ACPCore API is first called with nil, in addition to the push preferences change optin to optout or output to optin.
```

Comment:
feels a little wordy.  an alternative: 

    Push notification preferences (`a.push.optin`) are now forwarded to Analytics whenever the value passed to `setPushIdentifier` is different than the previous time it was called.